dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_postgresql.sql

```diff
@@ -489,6 +489,8 @@ CREATE TABLE t_ds_process_task_relation_log (
 
 create index process_task_relation_log_idx_project_code_process_definition_code on t_ds_process_task_relation_log (project_code,process_definition_code);
 
+create index idx_process_code_version on t_ds_process_task_relation_log (process_definition_code, process_definition_version);
```

Comment:
miss this ddl in mysql, please take a check.

Reply:
The index added.